clang analyzer failing

```
Run sudo pip3 install codechecker
  sudo pip3 install codechecker
  # disable IPO to make compilation faster
  cmake -S . -B build -DBUILD_TESTING=OFF -DCMAKE_INTERPROCEDURAL_OPTIMIZATION=OFF -DQD_ENABLE_ASSERTIONS=ON -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DENABLE_WARNING_ERROR=OFF
  # perform build so that generated sources get generated
  cmake --build build
  CodeChecker analyze build/compile_commands.json -o ./reports
  # return code 2 seems to mean issues were found, return code 3 means some checkers failed to run
  # print readable textual output of results
  CodeChecker parse --print-steps ./reports || true
  CodeChecker parse --print-steps --export html --output report_html ./reports || true
  shell: /usr/bin/bash -e {0}
  env:
    CC: clang
    CXX: clang++
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.
    
    If you wish to install a non-Debian packaged Python application,
    it may be easiest to use pipx install xyz, which will manage a
    virtual environment for you. Make sure you have pipx installed.
    
    See /usr/share/doc/python3.12/README.venv for more information.

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
Error: Process completed with exit code 1.
```

https://github.com/skupperproject/skupper-router/actions/runs/11332828372/job/31515770760#step:9:37

--- a/.github/workflows/clang-analyzer.yml
+++ b/.github/workflows/clang-analyzer.yml
@@ -74,7 +74,7 @@ jobs:
     # https://clang-analyzer.llvm.org/command-line.html
     - name: Create compilation database
       run: |
-        sudo pip3 install --break-system-packages codechecker
+        PIP_BREAK_SYSTEM_PACKAGES=1 sudo pip3 install codechecker
         # disable IPO to make compilation faster
         cmake -S . -B build -DBUILD_TESTING=OFF -DCMAKE_INTERPROCEDURAL_OPTIMIZATION=OFF -DQD_ENABLE_ASSERTIONS=ON -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DENABLE_WARNING_ERROR=OFF
         # perform build so that generated sources get generated

